invenio_rdm_records/requests/community_invitation.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Community submission request."""
+
+from flask_babelex import lazy_gettext as _
+from invenio_requests.customizations import DefaultRequestType
+
+
+#
+# Request
+#
+class CommunityInvitation(DefaultRequestType):
```

Comment:
Shouldn't this be in `invenio-communities`?

Reply:
I think it shouldn't.
This is a not part of the logic of a community, is just a request type that just happens to be related to a community.
Maybe it could go into a invenio-requests folder, but, as far as I understand, the non default request types are defined by the package needs the new type of request, and existence of this requests folder in invenio_rdm_records seems to point in that direction.

Reply:
I also think it should be in the communities. Like you said, it is related to community logic, and it does not refer to `rdm_records`, so in my opinion, it should belong to invenio-communities.